test/views/wallet/TransactionView/SendMobPanel/SendMobPanel.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+
+import SendMobPanel from '../../../../../app/views/wallet/TransactionView';
+import renderSnapshot from '../../../../renderSnapshot';
+
+function setupComponent() {
+  renderSnapshot(
+    <SendMobPanel />,
+  );
+
+  const sendQuery = screen.queryByText(/Please enter the amount of MOB you want to send and the public address of the recipient./i);
+  const acctName = screen.queryByText('Account Name');
+  const acctBalance = screen.queryByText('Account Balance');
+  const recipientInput = screen.queryByText('Recipient Public Address');
+
+  return {
+    acctBalance,
```

Comment:
Random nit. I try to discourage abbreviations -- they can make searching for things a bit harder if the searcher doesn't know about the one-off abbreviation. But it's fine in this test.